docs/fast_data/inputs_and_outputs.md

```diff
@@ -83,100 +98,108 @@ Delete operation:
 {
   "key": {
     "USER_ID": 123,
-    "FISCAL_CODE": "ABCDEF12B02M100O"
+    "TAX_CODE": "ABCDEF12B02M100O"
   },
   "value": null,
   "timestamp": "1234556789",
-  "offset": "100"
+  "offset": 100
 }
 ```
 
 #### Oracle Golden Gate
 
 ```yaml
-asyncapi: 2.4.0
+asyncapi: 2.6.0
 info:
-  title: Golden Gate Data Change Producer
+  title: Data Change API
   version: "1.0.0"
 channels:
-  GoldenGateDataChangeChannel:
+  GoldenGate:
     publish:
       message:
-        name: golden gate data change
+        name: Golden Gate data change message
         payload:
           type: object
-          additionalProperties: false
+          required:
+            - key
+            - value
+            - timestamp
+            - offset
           properties:
-            key: {}
+            key: 
+              type: string
+              description: String of the primary keys values joined by underscores
+              examples:
+                - pkValue1_pkValue2
             value:
               type: object
               additionalProperties: false
+              required:
+                - op_type
+                - pos
               properties:
                 op_type:
                   type: string
-                  enum: ["I", "D", "U"]
+                  description: Operation type. Can be *insert*, *update* or *delete*
+                  enum:
+                    - I
+                    - U
+                    - D
                 before:
                   type: object
+                  description: Whole record **before** the changes were applied (including the primary and foreign keys). This field is not defined with insert operation
                   additionalProperties: true
                 after:
                   type: object
+                  description: Whole record **after** the changes were applied (including the primary and foreign keys). This field is not defined with delete operation
                   additionalProperties: true
```

Comment:
Put if then else based on the op_type

Reply:
I believe it would complicate the readability more than being helpful, also other adapter don't have the op_type property so in the sake of coherence I'll leave it like that.